utils/roles/cluster_preperation/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+#  Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+---
+
+- name: Intialize provision_password_status
+  ansible.builtin.set_fact:
+    provision_password_status: true
+
+- name: Set provision_password_status to true
+  ansible.builtin.set_fact:
+    provision_password_status: true
+    provision_password: "{{ hostvars['127.0.0.1']['provision_password'] }}"
+  when:
+    - hostvars['127.0.0.1']['provision_password'] is defined
+    - hostvars['127.0.0.1']['provision_password'] | length > min_length | int - 1
+
+- name: Set passwordless SSH
+  when: provision_password_status
+  block:
+    - name: Set Facts
+      ansible.builtin.set_fact:
+        ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+        ssh_key_status: false
+
+    - name: Disable host key checking
+      ansible.builtin.replace:
+        path: /etc/ssh/ssh_config
+        regexp: '#   StrictHostKeyChecking ask'
+        replace: 'StrictHostKeyChecking no'
+      register: disable_host_key_checking
+      connection: local
+
+    - name: Disable strict mode checking
+      ansible.builtin.replace:
+        path: /etc/ssh/ssh_config
```

Comment:
/etc/ssh/ssh_config we can use as a variable

Reply:
corrected